ap.md

```diff
@@ -0,0 +1,256 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+<div class="large-12 columns">
+# Build Your Own Internet
+</div>
+<div class="large-7 small-12 columns">
+
+As explained in [wifi tutorial](/wifi.html), Tessel 2 can easily connect to wifi networks using the command line tool. If we're using a device connected to the same network as the Tessel, then we can connect to it through our device's web browser. Like most network-connected devices, communication over the web depends on a shared router, or access point, to direct requests and responses between every device on the network or through the Internet. This is fine and dandy if we want to use Tessel in a place with an available network, but what happens when that's not possible? 
+
+</div>
+<div class="large-5 small-12 columns right">
+![router](./images/router.png)
```

Comment:
Can we simplify this? Let's lose the wired computer and the extra printer, for example. And any way to use a T2 Fritzing diagram to represent the Tessel?